KarhooUISDK/Assets/Assets.xcassets/QuoteListFlister icons/u_baby-carriage.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "filename" : "u_baby-carriage.svg",
```

Comment:
Same issue with icons having the same name as the files. Could you please align them to our naming conventions?